baywatch.install

```diff
@@ -196,3 +197,24 @@ function baywatch_update_8005() {
     }
   }
 }
+
+/**
+ * Enable Tide OAuth module.
+ */
+function baywatch_update_8006() {
+  $private = 'private://';
+  \Drupal::service('file_system')->prepareDirectory($private, FileSystemInterface::CREATE_DIRECTORY);
+
+  vicgovau_core_install_module('tide_oauth');
+
+  $consumers = \Drupal::entityTypeManager()->getStorage('consumer')
+    ->loadByProperties([
+      'label' => 'Editorial Preview',
```

Comment:
I think this has to change now. It's not labelled `Editorial Preview` and needs the machine name to find the one to change.

Reply:
Done 👍